core/src/test/scala/unit/kafka/metrics/MetricsTest.scala

```diff
@@ -79,6 +79,17 @@ class MetricsTest extends KafkaServerTestHarness with Logging {
     assertFalse("Topic metrics exists after deleteTopic", checkTopicMetricsExists(topic))
   }
 
+  @Test
+  def testBrokerTopicMetricsAfterDeletingTopic() {
```

Comment:
@hachikuji I have renamed the method. Adding and deleting topics does test add/remove partitions as well. It is slightly harder to explicitly test partitions since partitions cannot be removed from topics.